prow/plugins/approve/approve.go

```diff
@@ -117,9 +118,9 @@ func helpProvider(config *plugins.Configuration, enabledRepos []string) (*plugin
 		var opts *plugins.Approve
 		switch len(parts) {
 		case 1:
-			opts = optionsForRepo(config, repo, "")
+			opts = orgOptions(config, repo)
```

Comment:
`enabledRepos` won't contain orgs so this case will never be reached. `enabledRepos` is a list of `org/repo` strings (no `org` strings).

Reply:
@cjwagner so should I open another PR to fix the same mistake in these places:
prow/plugins/blockade/blockade.go#L67
prow/plugins/dco/dco.go#L75
prow/plugins/lgtm/lgtm.go#L74
prow/plugins/slackevents/slackevents.go#L63
prow/plugins/trigger/trigger.go#L52
prow/plugins/welcome/welcome.go#L58

Reply:
Also, it goes against this commit: https://github.com/kubernetes/test-infra//commit/6e17de19d30ea12cc8a0a706956fc770d5c918eb

Reply:
The blockade plugin looks right, but the others are wrong. That commit is also wrong for the same reasons. Here is the relevant plugin help code used to generate the map from ```plugin name -> slice of `org/repo`s that the plugin is enabled on```: https://github.com/kubernetes/test-infra/blob/e3ca7a9f2e6b1a7aa2841b6f8359e6452e5f1a7c/prow/pluginhelp/hook/hook.go#L231-L262

Notice that any `org`s are flattened into `org/repo`s.

Reply:
Thanks, I will then make a PR to solve this mess... and then rebase this one.

Reply:
@cjwagner can you double check... because this was validated by @stevekuznetsov one year ago: https://github.com/kubernetes/test-infra/issues/10714#issuecomment-464308696

Reply:
I'm pretty confident about this. The snippet above plus these two other snippets show that the helpProvider is passed a list or repos (any orgs are flattened to repos first). 
https://github.com/kubernetes/test-infra/blob/e3ca7a9f2e6b1a7aa2841b6f8359e6452e5f1a7c/prow/pluginhelp/hook/hook.go#L151
https://github.com/kubernetes/test-infra/blob/e3ca7a9f2e6b1a7aa2841b6f8359e6452e5f1a7c/prow/pluginhelp/hook/hook.go#L86

If we are somehow including orgs in the list then this is a bug. The orgs are flattened into repos first so that any plugins that need to know all the repos don't have to look it up themselves. We could have each plugin flatten the list of orgs to repos themselves, but that would be wasteful. The other reason that the flattening is kind of useful is that it ensures we get a plugin help entry for each specific repo on the front end. Consider an org `o` with `lgtm` enabled, a repo `o/r1` that additionally has `approve` enabled, and a repo `o/r2` that doesn't enable anything else at the repo level. Without flattening the lists this would result in plugin help entries for `o` and `o/r1`, but not `o/r2` which is confusing to users.